inst/stan/epinowcast.stan

```diff
@@ -84,12 +84,16 @@ transformed parameters{
   vector[t] imp_obs[g]; // Expected final observations
   real phi; // Transformed overdispersion (joint across all observations)
   // calculate log mean and sd parameters for each dataset from design matrices
+  profile("parametric delay reference date"){
+  profile("parametric delay reference date - predictor"){
```

Comment:
Maybe you can have a look again - I changed the approach again. Now there is a prefix for the corresponding stan block ("transformed", "model", "generated") and the names are more like variables. I find this sensible because the profiling statements end up in a `data.frame` with one column giving these names. If we want to filter or pivot this dataframe, it is nice to have "variable-like" names instead of sentence case names or names with spaces I think...

Reply:
Good idea and I agree makes more sense when considering using this output.